test/test-fs.c

```diff
@@ -3284,6 +3284,62 @@ TEST_IMPL(fs_scandir_early_exit) {
 }
 
 
+TEST_IMPL(fs_openat) {
```

Comment:
Can you add some additional tests. For example, verifying that the async version works. There are also some tests for bad inputs that should probably include this new function.

Reply:
Yes. Will definitely do that. Just wanted to start a draft so we can know if someone thinks this definitely should not be added to libuv.